processor/src/operations/decorators/debug_tests.rs

```diff
@@ -27,6 +27,7 @@ fn test_print_all(logger: &mut Logger) {
     let stack = (1..=16).collect::<Vec<_>>();
     let inputs = ProgramInputs::new(&stack, &[], vec![]).unwrap();
     let mut process = Process::new(inputs);
+    process.advance_clock();
```

Comment:
Why do we need to add this line here and on line 44 as well?? I thought that this part of the could would be unaffected (since it relies only on `Memory::get_values()`).

Reply:
removed